rocAL/rocAL/source/meta_data/tf_meta_data_reader_detection.cpp

```diff
@@ -171,6 +171,7 @@ void TFMetaDataReaderDetection::read_record(std::ifstream &file_contents, uint f
     for(int i = 0; i < size_b_xmin; i++)
     {
```

Comment:
move this to previous line

Reply:
This change is part of the cleanup PR #38